account.py

```diff
@@ -183,7 +183,9 @@ def check_modify(cls, lines, modified_fields=None):
         '''
         Check if the lines can be modified
         '''
-        if modified_fields is None:
+
+        if (modified_fields is not None
+                and modified_fields <= cls._check_modify_exclude):
```

Comment:
This same check is done in the super, so I understand that it's not necessary to duplicate.

Reply:
It is true that we make the same check we do in the super, but if we don't check the same, we can delete or modify the move line in cases where the super would not let do it.

Maybe we need to delete the "check_modify" function in this module?